source/pages/SwitchNetwork/NetworkList.tsx

```diff
@@ -53,7 +52,7 @@ export const NetworkList = ({ isChanging }: { isChanging: boolean }) => {
 
   const handleChangeNetwork = async (network: INetwork, chain: string) => {
     try {
-      store.dispatch(setOpenDAppErrorModal(false));
+      wallet.setOpenDAppErrorModalState();
```

Comment:
is this change correct?

Reply:
As we said in the chat, yes, the function that is marked is correct and does not expect any arguments

Reply:
It is not correct. You actually created a function that already existed. Previously we were passing **false** as the argument of the function and now you are passing true.